revpi_hat/__init__.py

```diff
@@ -0,0 +1,3 @@
+from revpi_hat.hat import RevPiHatEEPROM, RevPiHatEEPROMAttributeException
+
+HAT_EEPROM_PATH = '/proc/device-tree/hat'
```

Comment:
As I don't have any clue about python. Is this a global constant? If yes why are you not using it later?

Reply:
Good catch, relic from development. I will remove it